conf/default.yaml

```diff
@@ -1,8 +1,20 @@
+debug: true
+s3:
+  Bucket: '{{ENV.MEDIAMANAGER_BUCKET}}'
+  SecretAccessKey: '{{ENV.MEDIAMANAGER_SECRETKEY}}'
+  AccessKeyId: '{{ENV.MEDIAMANAGER_ACCESSKEY}}'
+  Region: 'us-east-1'
```

Comment:
Confi is actually throwing a yaml parse error when I do {{s3.region}} there, not sure why.